website/static/playground.js

```diff
@@ -238,6 +238,8 @@ function replaceHash(hash) {
 
 function getCodemirrorMode(options) {
   switch (options.parser) {
+    // TODO: Remove the "postcss" case when prettier@>1.7.0 is released.
```

Comment:
Is this a prerelease task, or a postrelease task? (Or doesn't matter)

Reply:
post